### ISSUE TITLE
BUG: Transclusions with Insert Any File disappears right away from the drawing

### ISSUE THREAD

**Issue by lsolesen** [zsviczian/obsidian-excalidraw-plugin#1851]
Help me help you. I am a one man show doing this plugin as a part time hobby. There is no point in flooding me with issues, if there are too many, and they are poorly documented, I will just ignore them. Sorry...

Before creating a bug report, please
1. review recent release notes - maybe there is already an answer,
2. search issues (including closed ones) to see if there is anything similar.

⚠️ I will have to close all recorded bugs that do not provide this background information. Sorry, I need to control my workload/time. ⚠️

--------

**Your environment**
Please run `Command Palette/Show Debug info` in Obsidian and paste the result here.

SYSTEM INFO:
	Obsidian version: v1.6.3
	Installer version: v1.4.16
	Operating system: Windows 10 Pro 10.0.22631
	Login status: not logged in
	Insider build toggle: off
	Live preview: on
	Base theme: adapt to system
	Community theme: Things v2.1.13
	Snippets enabled: 0
	Restricted mode: off
	Plugins installed: 26
	Plugins enabled: 21
		1: Zotero Integration v3.1.8
		2: Obsidian_to_Anki v3.6.0
		3: Omnivore v1.10.3
		4: Tasks v7.4.0
		5: Periodic Notes v0.0.17
		6: Calendar v1.5.10
		7: Dataview v0.5.67
		8: Media Extended v3.2.4
		9: Timestamp Notes v1.0.8
		10: Templater v2.3.3
		11: ReadItLater v0.4.0
		12: QuickAdd v1.9.0
		13: Kindle Highlights v1.9.2
		14: Spaced Repetition v1.12.4
		15: Excalidraw v2.2.7
		16: Toggl Track v0.11.0
		17: ExcaliBrain v0.2.15
		18: Charts View v1.2.7
		19: Projects v1.17.3
		20: Advanced Slides v1.20.0
		21: Beautitab v1.6.1

RECOMMENDATIONS:
	Custom theme and snippets: for cosmetic issues, please first try updating your theme and disabling your snippets. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community theme and snippets.
	Community plugins: for bugs, please first try updating all your plugins to latest. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community plugins.

**Describe the bug**

When I Insert Any File, it briefly shows, but then the content disappears again right after. That essentially means I cannot transclude anything into the drawings. I tried with images, notes, other excalidraw drawings, but it happens every time. I tried with a completely new drawing, and old drawings that I was able to put things into earlier. As far as I can tell, I have not done anything except keeping Excalidraw up to date.

**To Reproduce**

What I do is just click on "Insert Any File", however, since this is so essential to Excalidraw, I am pretty sure, that you cannot see the error on your side at the moment. As I was really want this to work, however, if you can guide me to give you the correct info for a fix, I would very much appreciate this.

**Expected behavior**

**Screenshots**

https://app.screencastify.com/v3/watch/mzcW0I6jKDID3N79tmEh

**Additional context**

**Comment by zsviczian** [comment]
This feels like a plugin automatically updating the Excalidraw file in the background. Can you reproduce this when Only Excalidraw is enabled?

**Comment by lsolesen** [comment]
You are right, something is updating the Excalidraw file. I disabled all community plugins, and it still happens. However, I digged a little deeper.

Starting with the Presentation - Fysiologi file:

![Obsidian_qK6o5RVzQj](https://github.com/zsviczian/obsidian-excalidraw-plugin/assets/148026/8fec9dfa-01a0-44eb-9e07-57c2dcd20b30)

I insert any file. That worked. However, an additional file is created, so I now have two files. Both the Presentation - Fysiologi and a replica Excalidraw file.

![Obsidian_R55PiGuIyy](https://github.com/zsviczian/obsidian-excalidraw-plugin/assets/148026/d26bf178-abc8-4789-870a-0368081fb49d)

Trying to insert another image in the original file, but now it disappears as in the video.

But inspecting the Excalidraw file, not the image is there.

![Obsidian_7jZOGRD58H](https://github.com/zsviczian/obsidian-excalidraw-plugin/assets/148026/4d746481-9e77-4ce4-8ae5-30e48a1558f7)

The original file was originally a markdown, where I used the Convert markdown note to Excalidraw Drawing.

This has happened on all of the files where I tried to make it work with backside and frontside of card.

**Comment by lsolesen** [comment]
This was done with this:

SYSTEM INFO:
	Obsidian version: v1.6.3
	Installer version: v1.4.16
	Operating system: Windows 10 Pro 10.0.22631
	Login status: not logged in
	Insider build toggle: off
	Live preview: on
	Base theme: adapt to system
	Community theme: Things v2.1.13
	Snippets enabled: 0
	Restricted mode: off
	Plugins installed: 25
	Plugins enabled: 2
		1: Excalidraw v2.2.7
		2: ExcaliBrain v0.2.15

RECOMMENDATIONS:
	Custom theme and snippets: for cosmetic issues, please first try updating your theme and disabling your snippets. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community theme and snippets.
	Community plugins: for bugs, please first try updating all your plugins to latest. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community plugins.

**Comment by lsolesen** [comment]
I figured out the problem. I was the setting Autoexport as Excalidraw.

![image](https://github.com/zsviczian/obsidian-excalidraw-plugin/assets/148026/f4979012-f6fe-4f2b-81cb-16acb749bd07)

Turning this off, made everything work as expected.
